state/backups/restore.go

```diff
@@ -276,10 +276,15 @@ var sshCommand = ssh.Command
 
 // runViaSSH runs script in the remote machine with address addr.
 func runViaSSH(addr string, script string) error {
-	// This is taken from cmd/juju/ssh.go there is no other clear way to set user
```

Comment:
Why? ssh.Client.Command documents the argument as taking the syntax [user@]host. It shouldn't be needed.